.github/workflows/qa-labels.yml

```diff
@@ -1,6 +1,6 @@
 name: Add QA labels to Elastic Agent issues
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
As there are some security risks attached to use `_target` I looked a bit more into this change. Based on line 5 this runs always when closed (not only merged) but based on line 12, it is only run when it is actually merged? And merging can only be done by members of the team so all should be good?